content/2-templating/6-conditional/ripple/TrafficLight.ripple

```diff
@@ -0,0 +1,26 @@
+import { track } from "ripple";
+
+const TRAFFIC_LIGHTS = ["red", "orange", "green"];
+
+export component TrafficLight() {
+  let lightIndex = track(0);
+
+  const light = track(() => TRAFFIC_LIGHTS[@lightIndex]);
+
+  function nextLight() {
+    @lightIndex = (@lightIndex + 1) % TRAFFIC_LIGHTS.length;
+  }
+
+  <button onClick={nextLight}>{"Next light"}</button>
+  <p>{`Light is ${@light}`}</p>
+  <p>
+    {"You must "}
+    if (@light === "red") {
+      <span>{"STOP"}</span>
+    } else if (@light === "orange") {
+      <span>{"SLOW DOWN"}</span>
+    } else if (@light === "green") {
+      <span>{"GO"}</span>
+    }
```

Comment:
```suggestion
    switch (@light) {
      case "red":
        <span>{"STOP"}</span>
      case "orange":
        <span>{"SLOW DOWN"}</span>
      case "green":
        <span>{"GO"}</span>
    }
```

Reply:
Okay, thanks!
I couldn't find the tracker for it in https://github.com/trueadm/ripple/issues